test/component/applicant/testAddress.js

```diff
@@ -1,14 +1,14 @@
 'use strict';
 
 const TestWrapper = require('test/util/TestWrapper');
-// const ExecutorsNumber = require('app/steps/ui/executors/number/index');
+const DeceasedName = require('app/steps/ui/deceased/name/index');
 const testAddressData = require('test/data/find-address');
 const formatAddress = address => address.replace(/\n/g, ' ');
 const testHelpBlockContent = require('test/component/common/testHelpBlockContent.js');
 
 describe('applicant-address', () => {
     let testWrapper;
-    // const expectedNextUrlForExecsNumber = ExecutorsNumber.getUrl();
+    const expectedNextUrlForExecsNumber = DeceasedName.getUrl();
```

Comment:
Change name of const to `expectedNextUrlForDeceasedName` and replace all occurrences in this file.